content/applications/sales/crm/optimize/utilize_activities.rst

```diff
@@ -159,67 +161,92 @@ past due activities.
 The pipeline is most effective when it is kept up-to-date and accurate to the interactions it is
 tracking.
 
+.. _crm/activity-plans:
+
 Activity plans
---------------
+==============
+
+*Activity Plans* are preconfigured sequences of activities that can be triggered with one click,
+and replicated with little to no manual setup. Activity plans can be customized for various
+workflows, with multiple deadlines and responsible users assigned in a single plan.
```

Comment:
Consider condensing this first paragraph to necessary and specific information. Seems a bit wordy, borderline sales-y.